code/src/lib/Button.js

```diff
@@ -0,0 +1,42 @@
+import styled from 'styled-components'
+
+export const Button = styled.button`
+  padding: 8px;
+  margin-left: 36px;
+  background: ${(props) => props.background || '#6F6FE6'};
```

Comment:
Can't see that you're passing down any props with background-color for any styled component. Are the buttons supposed to have different colours at different conditions? Otherwise, just keep the hex-colours :)